src/functions-and-arrays.js

```diff
@@ -52,16 +162,39 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
-
+function uniquifyArray(someArray) {
+  let uniqueArray = [];
+
+  if (someArray.length === 0) {
+    return null;
+  }
+  
+  for (let i = 0; i < someArray.length; i++) {
+    if (uniqueArray.includes(someArray[i]) == false) {
+      uniqueArray.push(someArray[i]);
+    } else if (uniqueArray.includes(someArray[i]) == true) {
```

Comment:
This check on line 175 is redundant, since it is the exact opposite of line 173. In these cases, best to use a simple `if else` instead! ✌🏻